### PR TITLE
Mining Update Part 1

### PR DESCRIPTION
**Pull request by ArcLumin**
The first part of my mining update

Adds in loads of new items to the mining rewards vendor
Boosts iron from 1 point to 5 points
Tweaked some existing rewards
Buffed mining drones and stimpacks

### CODE REVIEW

**Comment by Pyko1** [comment]
You, you monster! You removed whiskey!

**Comment by Swagile** [comment]
slamming whiskey over your face and lighting yourself on fire is a time honored shaft miner tradition

you absolute madman

**Comment by ArcLumin** [comment]
Might as well readd whiskey for like

10 points

**Comment by Scinexx** [comment]
Thank god, finally makes mining worth getting more than 5k points and giving up. Half the time I just mined for stimpacks then looked for crates.

**Comment by ghost** [comment]
I don't like giving easy access to night vision.
Also having a fully functioning ripley just vend out of a machine is silly. It would probably be better if it was individual parts.

**Comment by ArcLumin** [comment]
Mining rewards are teleported in. If it was just the parts, they'd need the tools, the plasteel, etc, which is a huge pain, especially for that price. Also, night vision is useful as fuck on mining. I can boost the price, however

**Comment by ghost** [comment]
Oh yeah, that's okay then.
Price needs to be higher for night vision goggles. They're very useful.

**Comment by ArcLumin** [comment]
Doubled price on gar night vision mesons

**Comment by ghost** [comment]
I'm skeptical but I'm willing to see this trialed. +1 for now.

**Comment by TDcoolguy300** [comment]
Code looks fine mate

**Comment by ArcLumin** [comment]
Doubled stimpack prices due to the buffs. At least now they aren't worthless

**Comment by SplergeBucket** [comment]
we can see how it goes

**Comment by GuyonBroadway** [comment]
My only "mild" concern is the ability for miners to get a usually only Deathsquad/MemeOps item (that has never been purchased) but at 15000 points that's probably OK.  Else it'd be nice if miners could buy something that allows science to get to bluespace 10 so they can build plenty of these teleporters for them.

**Comment by ArcLumin** [comment]
The amount of power it uses for a mediocre power is insane. It'll mostly be used for science bluespace 10

**Comment by ghost** [comment]
I disagree with night vision price being increased. Night vision isn't all that broken when compared to thermals or xray, and with this new increased price, miners can get their hand on night vision faster bugging science for them. Night vision also ducks up your eyes if you forget to take it off before using a welding tool, which you need to do to repair your ripley.

**Comment by Swagile** [comment]
I tried so hard
And got so far
But in the end
It doesn't even matter
I had to fall
To lose it all
But in the end
It doesn't even matter







tl;dr heads get in here boyos

**Comment by ghost** [comment]
Considering looping said "we can see how it goes" that's a +1.

**Comment by ArcLumin** [comment]
Approval done, merging

**Comment by LuisReyesX** [comment]
inb4 RnD powercreep

**Comment by Swagile** [comment]
rnd already doesn't have half the time to do the cool shit / use the cool shit as traitor as there is a dozen better alternatives that traitors can spawn.

let em have their fun for once considering the short round lengths.

**Comment by ArcLumin** [comment]
This is more mining powercreep than science lad. In fact, some of the science exclusivity is gone with this
